server/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+name: whatsync
+services:
+  server:
+    build:
+      context: ..
+      dockerfile: server/Dockerfile
+
+    container_name: whatsync-server
+    env_file:
+      - .env
+    environment:
+      - RUNNING_IN_DOCKER=true
+    hostname: whatsync-server
+    ports:
+      - 8080:8080
+    restart: unless-stopped
```

Comment:
I don't think there's a need for 3 separate `docker-compose.yml`. How about just keeping the main one?